html/example.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
+<head>
+  <meta charset="utf-8" />
+  <meta name="generator" content="pandoc" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
+  <title>example</title>
+  <style type="text/css">
```

Comment:
What is this file for? Can it not use in-file CSS?

Reply:
It's just pandoc's default. We'll fix it later.

The commit is really just supposed to be for a basic python script.